qml/pages/VenueMapPage.qml

```diff
@@ -48,7 +48,7 @@ BVApp.Page {
 
             onClicked: {
                 var query = venueCoordinate.latitude + "," + venueCoordinate.longitude
-                if (BVApp.Theme.isIos || BVApp.Theme.isOSX) {
+                if (BVApp.Platform.isIos || BVApp.Platform.isMacOS) {
                     Qt.openUrlExternally("https://maps.apple.com/?q=" + query)
```

Comment:
As I already told you IRL: I totally see your point of staying close with v-play upstream. The main reason for me moving these ```isMyHottestSystem()``` questions to a ```Platform``` object was that I assumed the semantics of our questions to differ a bit from the v-play ones: 

With v-play, iOS and Android are actually themes you can set. Of course, by default on Android the Android theme is selected and on iOS the iOS theme, but you can also switch to the iOS Theme on Android if you like, or...

My assumption was that we are going to be adding a ```isSailfishOS()``` question, which would change the semantics of our questions a bit. 

So much for the explanation why I made this change. In the end, I don't care that much ;).